core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -339,9 +354,9 @@ private void loadQueueForMediaSession() {
             }
             emitter.onSuccess(queueItems);
         })
-                .subscribeOn(Schedulers.io())
-                .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(queueItems -> mediaSession.setQueue(queueItems), Throwable::printStackTrace);
+        .subscribeOn(Schedulers.io())
+        .observeOn(AndroidSchedulers.mainThread())
+            .subscribe(queueItems -> mediaSession.setQueue(queueItems), Throwable::printStackTrace);
```

Comment:
Please revert to keep the git history clean. Note that CI only checks lines that are changed, so it is okay if these lines do not follow the code style.